version.tf

```
@@ -14,5 +14,9 @@ terraform {
       source  = "IBM-Cloud/ibm"
       version = ">= 1.71.2, < 2.0.0"
     }
+    random = {
+      source  = "hashicorp/random"
+      version = "3.7.2"
```

As per the code comment above, best practise is to use a range in modules like so:
```suggestion
      version = ">= 3.7.2, <4.0.0"
```

---

after discussions on call, there is no module here. So actually all the versions in this file should be locked to the latest version as a best practise for DAs